icm-injection-scripts/scripts/inject-icm.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# Inject an ICM (image content manifest) file with content sets for backwards compatibility
+#
+# https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/content_manifest.json
+#
+# This is not a file we want to inject always into the future, but older Red
+# Hat build systems injected a file like this and some third-party scanners
+# depend on it in order to map rpms found in each layer to CPE ids, to match
+# them with vulnerability data. In the future, those scanners should port to
+# using the dnf db and/or SBOMs to make that same match. Consider this
+# deprecated.
+#
+# This is only possible for images built hermetically with prefetch
+
+set -euo pipefail
+
+IMAGE="${1}"
+SQUASH="${SQUASH:-false}"
+
+icm_filename="content-sets.json"
```

Comment:
OSBS generates `<name>-<version>-<release>.json` name to avoid overwritting ICMs of parent images. Are we ok with overwriting parent images ICMs here?

Reply:
Yeah, I asked Clair devs and they said it's ok. They're always looking at things layer by layer anyways - they only consult the ICM that happens to appear in the same layer they're considering.